src/System.Private.ServiceModel/tests/Scenarios/Security/TransportSecurity/Https/HttpsTests.cs

```diff
@@ -78,31 +84,38 @@ public static void Https_SecModeTrans_ClientCredTypeNone_ServerCertValModePeerTr
         }
     }
 
-    // This test is expected to validate the service certificate using PeerTrust with X509CertificateValidationMode set to PeerOrChainTrust.
 #if FULLXUNIT_NOTSUPPORTED
     [Fact]
     [ActiveIssue(959)] // Server certificate validation not supported in NET Native
 #else
     [ConditionalFact(nameof(Root_Certificate_Installed),
                      nameof(Client_Certificate_Installed),
+                     nameof(Peer_Certificate_Installed),
                      nameof(SSL_Available))]
 #endif
     [OuterLoop]
-    public static void Https_SecModeTrans_ClientCredTypeNone_ServerCertValModePeerOrChainTrust_EchoString()
+    // Asking for PeerTrust alone should throw SecurityNegotiationException
+    // if the certificate is not in the TrustedPeople store.  For this test
+    // we use a valid chain-trusted certificate that we know is not in the
+    // TrustedPeople store.
+    public static void Https_SecModeTrans_CertValMode_PeerTrust_Fails_Not_In_TrustedPeople()
```

Comment:
No -- this is simply a negative test to demonstrate that asking for PeerTrust only will fail if the cert is not in TrustedPeople.  From the standpoint of these tests, this exercises the code path that rejects such certificates in the X509CertificateValidator.